src/lib/components/PaginationArrow.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+	import CarbonCaretLeft from "~icons/carbon/caret-left";
+	import CarbonCaretRight from "~icons/carbon/caret-right";
+
+	export let direction: "next" | "previous";
+	export let onClick: (dir: "next" | "previous") => void = () => {};
+	export let isDisabled = false;
+</script>
+
+<button
```

Comment:
This should be a link `<a>` to prev/next page (vs button with click event)

Reply:
handled in https://github.com/huggingface/chat-ui/pull/772/commits/53ed09bc5c6a17d95dcd844deeba44fd4f7482c0